pkg/iac/terraform/state/test/route/result.golden.json

```diff
@@ -105,5 +105,14 @@
   "VpcEndpointId": null,
   "VpcPeeringConnectionId": null,
   "Timeouts": null
+ },
+ {
```

Comment:
In theses state parsing tests, we need to have a minimal state with only resources related in the test.

It's my fault if we do not have a minimal state here, I will do a PR to remove `aws_vpc`, `aws_route_table` and `aws_internet_gateway` from the state to avoid this kind of unwanted impact.

I'm doing the PR asap so you can rebase this PR.

Reply:
Got it !